dali/kernels/slice/slice_flip_normalize_permute_pad_gpu.h

```diff
@@ -101,13 +102,27 @@ class SliceFlipNormalizePermutePadGpu {
     se.add<mm::memory_kind::pinned, detail::SampleDesc<Dims>>(num_samples);
     se.add<mm::memory_kind::device, detail::SampleDesc<Dims>>(num_samples);
 
+    int blocks_per_sm_;
+    CUDA_CALL(cudaOccupancyMaxActiveBlocksPerMultiprocessor(&blocks_per_sm_,
+      detail::SliceFlipNormalizePermutePadKernel<false, false, false, OutputType, InputType, Dims>, 
```

Comment:
Is there any difference (significant) if the occupancy is calculated for `detail::SliceFlipNormalizePermutePadKernel<false, false, false` but different variant is used in the bechmark?

Reply:
There is no difference for this call, it always returns 6 blocks per SM.

Reply:
Understood